kplc

```diff
@@ -5,6 +5,7 @@ shopt -s extglob
 
 debug=false
 verbose=false
+result_only=false
```

Comment:
This is a specific option (not for the whole file), so should be defined down lower at the options-handling section (line 92, where we define `backend` as well)